### PR TITLE
added page number option

### LINKED ISSUE #722: Add `show page numbers` in `Settings` section
Add the option added in https://github.com/Swati4star/Images-to-PDF/issues/702 to settings section also

### PR DESCRIPTION
**Pull request by Ni3verma**
# Description

added show page number option in settings also.
Screenshot_20190424-204846

Fixes #722 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Ni3verma** [comment]
@Swati4star @vyankatesh24 please review

File: app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java [hunk 1]
```diff
@@ -289,4 +293,57 @@ public void setTheme() {
...
+                    }
+                    if (cbDefault.isChecked ()) {
+                        editor.putString (Constants.PREF_PAGE_STYLE, style);
+                        editor.putInt (Constants.PREF_PAGE_STYLE_ID, id);
+                        editor.commit ();
+                    } else {
+                        editor.putString (Constants.PREF_PAGE_STYLE, null);
+                        editor.putInt (Constants.PREF_PAGE_STYLE_ID, -1);
+                        editor.commit ();
+                    }
```
Comment (sidhuparas) [inline app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java]: Why is there a space after `putString` and `putInt`?

**Comment by vyankatesh24** [comment]
@Ni3verma Upload Screenshots!

File: app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java [hunk 2]
```diff
@@ -137,6 +139,8 @@ public void onItemClick(int position) {
             case 6:
                 changeMasterPassword();
                 break;
+            case 7:
+                setShowPageNumber();
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java]: add break also

File: app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java (hunk 1 above)
Comment (Ni3verma) [reply app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java]: copied the code from image to pdf fragment. will fix it

**Comment by Ni3verma** [comment]
@Swati4star @vyankatesh24 @sidhuparas  added the changes, please review again

**Pull request merged**